examples/angular/src/environments/environment.development.ts

```diff
@@ -5,6 +5,6 @@ export const environment = {
    * for security purposes, make sure is read-only.
    * More info: https://www.dotcms.com/docs/latest/permissions
    */
-  authToken: 'YOUR_AUTH_TOKEN',
-  siteId: 'true',
+  authToken: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhcGk0ZjFmNjljZS04ZmZjLTRkZWQtOWU2Zi1jNTQ4ZjM3M2ZmNDQiLCJ4bW9kIjoxNzI0ODc3MDIyMDAwLCJuYmYiOjE3MjQ4NzcwMjIsImlzcyI6IjRhYTRiYjY2Y2QiLCJleHAiOjE3MjU3NDEwMjIsImlhdCI6MTcyNDg3NzAyMiwianRpIjoiM2ViYzc0OWMtN2JjMS00NmQ2LTgyNTUtZTQyOTE1YWVjMTM3In0.aCcXbdZlbGqnHggPpOsr6JUanC8T7WFxD5tg1GxpX74',
+  siteId: 'true'
```

Comment:
update the site id to something that has meaning

Reply:
Actuallt I have to clean this file and we need to fix ths siteId this.